### PR TITLE
Created vmm-reference-msrs.md

### PR DESCRIPTION
**Pull request by ArkaMandal2001**
Described working of msrs.rs

### CODE REVIEW

**Comment by ArkaMandal2001** [comment]
https://github.com/codenet/understanding-rust-vmm/pull/8

File: vmm-reference-msrs.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+Here is a brief description of how msrs.rs works from "vmm-reference/src/vm-vcpu-ref/src/x86_64/msrs.rs"
```
Comment (kshitij86) [inline vmm-reference-msrs.md:1]: Testing

File: vmm-reference-msrs.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+MSRS is used to store a half done process' state and continue with some other process and then come back to it, restore it and continue wth the work.
+
```
Comment (Laxman824) [inline vmm-reference-msrs.md:3]: Testing

**Review by Laxman824 — COMMENTED** [review]
Testing

File: vmm-reference-msrs.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+struct MsrRange {
```
Comment (CoderPanda1729) [inline vmm-reference-msrs.md:4]: Testing

**Review by priyanshs — COMMENTED** [review]
testing

**Review by kshitij86 — APPROVED** [review]

File: vmm-reference-msrs.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+Here is a brief description of how msrs.rs works from "vmm-reference/src/vm-vcpu-ref/src/x86_64/msrs.rs"
+MSRS is used to store a half done process' state and continue with some other process and then come back to it, restore it and continue wth the work.
```
Comment (codenet) [inline vmm-reference-msrs.md:2]: wth -> with

Comment (codenet) [reply vmm-reference-msrs.md:2]: Unclear description. I don't get what MSRS is by reading this description.

**Review by codenet — CHANGES_REQUESTED** [review]
Not a great PR. It doesn't go deep into what are MSRs, why are they used, how are they used in overall KVM functioning etc.

**Comment by codenet** [comment]
This PR describes the same topics as https://github.com/codenet/understanding-rust-vmm/pull/12/files Please directly add changes that you think are unique in your PR to the other PR.

**Pull request closed without merging**